vm/ops/store.cpp

```diff
@@ -317,6 +317,35 @@ void x86CPU::op_popaW(){
     SetReg(AX,Pop());
 }
 
+void x86CPU::op_insb_m8_dx(){
```

Comment:
Please move these to the string.cpp file. These are string instructions that can be used with REP. 

Also string instructions (ones that end with "s" usually) are weird. They reference memory but from a predetermined register set: "6C INS m8, DX | Input byte from I/O port specified in DX into memory location specified in ES:(E)DI." 

So it doesn't have an immediate operand in the bytecode but rather uses the DI/EDI register for the memory address. Check the other string instructions for guidance.